dataikuapi/dss/feature_store.py

```diff
@@ -0,0 +1,17 @@
+class DSSFeatureStore(object):
+    def __init__(self, client):
+        """
+        A handle on the Feature Store.
+        Do not create this class directly, use :meth:`DSSClient.get_feature_store`
+        """
+        self.client = client
+
+    def list_feature_groups(self):
```

Comment:
Since this method is cross-project, it cannot return dataset "names". It may possibly return dataset full names (project + dataset), but in that case, the returned value is fairly unusable.

Please make it return a list of DSSFeatureGroupsListItem which have:

* `.id` (returns full project+name)
* `.get_as_dataset()` which returns a DSSDataset from the correct project

Reply:
It returns indeed full ids, and the only use of those ids is to retrieve the associated dataset.
ok, will do.